app/src/main/java/com/example/triptracker/view/Navigation.kt

```diff
@@ -54,6 +55,7 @@ class Navigation(val navController: NavHostController) {
   fun navigateTo(destination: TopLevelDestination) {
     navController.navigate(destination.route) {
       currentDestination = destination
+      navController.currentBackStackEntry?.arguments?.putString("id", "")
```

Comment:
great fix for the back and forth problem. Could you add a quick comment to explain why you made this choice?